composeApp/src/commonMain/kotlin/romHandlers/Gen1RomHandler.kt

```diff
@@ -1,5 +1,8 @@
 package romHandlers
 
 import configurations.Gen1RomConfiguration
-class Gen1RomHandler(romConfiguration: Gen1RomConfiguration) : AbstractGBCRomHandler(romConfiguration) {
+import romHandlers.abstractRomHandlers.AbstractGBRomHandler
+
+class Gen1RomHandler(romConfiguration: Gen1RomConfiguration, rom: ByteArray)
```

Comment:
```suggestion
class Gen1RomHandler(romConfiguration: Gen1RomConfiguration, rom: ByteArrayRom)
```
Does it make more sense to have the wrapper as the parameter? Feels like we could add more functionality to the wrapper in the future if we wanted, versus the ByteArray interface.

Reply:
Same for the rest of the RomHandler classes